### PR TITLE
feat: added vuepress as blog engine

### PR DESCRIPTION
**Pull request by guacamole**
- Added Documentation, Getting started guide, gif, change-log

Signed-off-by: guacamole <gunjanwalecha@gmail.com>

### CODE REVIEW

File: docs/guide/about.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # About - OpenRegistry
```
Comment (jay-dee7) [inline docs/guide/about.md:7]: @guacamole can we please add frontmatter here too?

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Open Registry
```
Comment (jay-dee7) [inline docs/README.md:7]: @guacamole frontmatter here too 🙏🏻

File: docs/guide/changelog.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+---
+lang: en-US
+title: Changelog
+description: Changelog for OpenRegistry
+---
```
Comment (jay-dee7) [inline docs/guide/changelog.md:5]: @guacamole please add an extra space after the frontmatter ending line, also empty line under every heading element

File: docs/guide/contact.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Contact
```
Comment (jay-dee7) [inline docs/guide/contact.md:7]: @guacamole frontmatter here too

File: docs/guide/documentation.md [hunk 5]
```diff
@@ -1,48 +1,52 @@
-# Doumentation
+---
```
Comment (jay-dee7) [inline docs/guide/documentation.md:1]: @guacamole please use standard values for all markdown files

File: docs/guide/gettingStarted.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+# Getting Started
```
Comment (jay-dee7) [inline docs/guide/gettingStarted.md:7]: @guacamole empty lines under all headings here too

**Review by jay-dee7 — CHANGES_REQUESTED** [review]
@guacamole just requested some minor fixes, please go through them and after that we're good to merge this PR. great work 🥇 🥇 🥇

File: docs/guide/contact.md (hunk 4 above)
Comment (guacamole) [reply docs/guide/contact.md:7]: done

File: docs/guide/documentation.md (hunk 5 above)
Comment (guacamole) [reply docs/guide/documentation.md:1]: done

File: docs/guide/gettingStarted.md (hunk 6 above)
Comment (guacamole) [reply docs/guide/gettingStarted.md:7]: done

**Review by jay-dee7 — APPROVED** [review]
@guacamole LGTM 💯

**Pull request merged**